src/components/SettingsDialog/ExportBoard/PrintView/PrintView.scss

```diff
@@ -0,0 +1,215 @@
+@import "src/constants/style.scss";
+
+$note__border-radius: 16px;
+$border-radius--round: 50px;
+
+$color-backlog-blue-transparent: rgba(0, 87, 255, 0.25);
+$color-button: $color-lean-lilac;
+$color-button-shadow: rgba(133, 144, 147, 0.5);
+
+.print-view__portal {
+  background-color: rgba(0, 0, 0, 0.8);
+  backdrop-filter: blur(10px);
+}
+
+.print-view__button-container {
+  position: fixed;
+  top: 92%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.print-view__button {
+  background: lighten($color-button, 35%);
+  border: solid 1px lighten($color-button, 35%);
+  border-radius: $border-radius--round;
+  margin: $margin--small;
+  cursor: pointer;
+  font-weight: bold;
+  font-size: 16px;
+  padding: 3px 5px;
```

Comment:
There are better ways to handle icons than using random values to apply centering.

Due to the `icon-close.svg` having margin, it needs to be larger than the print icon, but it does not require arbitrary padding and margin values.